packages/generator-single-spa/src/svelte/templates/rollup.config.js

```diff
@@ -0,0 +1,70 @@
+import svelte from "rollup-plugin-svelte";
```

Comment:
Did you write this rollup config? What did you base it on?

Reply:
This config is based on https://github.com/sveltejs/template

Reply:
👍